src/applications/yellow-ribbon/components/SearchResultsPage/index.js

```diff
@@ -93,12 +94,10 @@ export const SearchResultsPage = ({
 );
 
 SearchResultsPage.propTypes = {
-  // From mapStateToProps.
   hasFetchedOnce: PropTypes.bool.isRequired,
   showMobileForm: PropTypes.bool.isRequired,
-  totalResults: PropTypes.number,
-  // From mapDispatchToProps.
   toggleShowMobileForm: PropTypes.func.isRequired,
+  totalResults: PropTypes.number,
 };
```

Comment:
I appreciate the alphabetization!

Reply:
Would love to take credit, but it's required by the checks lol